lib/pbench/server/api/resources/__init__.py

```diff
@@ -931,10 +953,13 @@ def authorize(self, params: ApiParams) -> Optional[ApiAuthorization]:
         Returns
             The values for username and access policy to use for authorization.
         """
-        if self.authorization == API_AUTHORIZATION.DATASET:
+        if self.authorization == API_AUTHORIZATION.ADMIN:
+            return ApiAuthorization(type=self.authorization, role=self.operation)
+        elif self.authorization == API_AUTHORIZATION.DATASET:
```

Comment:
Since it seems unlikely that `authorization` is `ADMIN`, do you want to check for that case last?